### ISSUE TITLE
Support for Crossref Grant ID

### ISSUE THREAD

**Issue by philippconzett** [IQSS/dataverse#7286]
In a recent report [(https://doi.org/10.29242/report.effectivedatapractices2020),](https://doi.org/10.29242/report.effectivedatapractices2020) the Association of Research Libraries (ARL) recommends wide adoption of these 5 core PIDs to power findability of research data:

![image](https://user-images.githubusercontent.com/21955790/94335172-10523d80-ffda-11ea-91bd-8cdedab43036.png)

I suggest that Crossref Grant IDs be implemented in the Citation Metadata section Grant Information.

**Comment by djbrooke** [comment]
Hey @philippconzett - thanks for reporting! Thanks also for sharing the ARL report.

@jggautier for this and #7285, do you have any thoughts about implementation?

**Comment by jggautier** [comment]
This is the Grant Information compound field as of Dataverse 5 (the fields come from DDI Codebook):

<img width="941" alt="94965723-fc22aa80-04c9-11eb-8a38-a5be72203dd0" src="https://user-images.githubusercontent.com/18374574/95208412-0ccb6d00-07b7-11eb-9c73-e46e8db60929.png">

Crossref Grant IDs are DOIs for grants. I'm assuming it would be important for Dataverse to send this metadata to DataCite (#2917, #5889). But it doesn't seem like the DataCite schema has a property for something like a Crossref Grant ID.

In Dataverse, the Grant Number field is mapped to DataCite's awardNumber property (and appears in Dataverse's OpenAIRE export, which is based on DataCite). awardNumber is defined as "The code **assigned by the funder** to a sponsored award (grant)." DataCite also has an awardURI property, "The URI leading to a page **provided by the funder** for more information about the award (grant)." They give `https://www.moore.org/grants/list/GBMF3859.01` as an example of an awardURI.

Neither of those two properties seem right. So I'm not sure if it would be appropriate to:
- add a new field for Crossref Grant IDs or 
- expect depositors to put Crossref Grant IDs in Dataverse's existing Grant Number field, maybe after changing the field label to something like "Grant Number or ID":

   <img width="779" alt="Screen Shot 2020-10-02 at 7 06 13 PM" src="https://user-images.githubusercontent.com/18374574/95213825-3a1b1980-07bd-11eb-9921-8c809f8db889.png">

[I asked in DataCite's Metadata Google Group](https://groups.google.com/g/datacite-metadata/c/I1dQzqn1EHs).

@philippconzett, have you seen Crossref Grant IDs used in other types of repositories? And would you happen to know where to find existing Crossref Grant IDs? I couldn't find any.

**Comment by jggautier** [comment]
> @philippconzett, have you seen Crossref Grant IDs used in other types of repositories? And would you happen to know where to find existing Crossref Grant IDs? I couldn't find any.

There are examples of Crossref Grant IDs are at https://www.crossref.org/documentation/research-nexus/grants. One of the examples is https://doi.org/10.37717/220020589. And we can see how that grant ID is represented in the Crossref APIs by visiting https://api.crossref.org/works?filter=award.number:220020589

**Comment by philippconzett** [comment]
@jggautier No, I can't say I have. Maybe we could reach out to some of the authors of the ARL report this issue originate from? See the reference at the top of the issue.

**Comment by jggautier** [comment]
Ha, hi @philippconzett. In my comment from two days ago I meant to answer my own question about examples of Crossref Grant IDs. Sorry for the confusion.
